DataFormats/EcalDigi/interface/EcalDigiCollections.h

```diff
@@ -39,10 +39,6 @@ class EBDigiCollection : public  EcalDigiCollection {
   EBDigiCollection(size_type istride=MAXSAMPLES) : 
     EcalDigiCollection(istride, EcalBarrel){}
   void swap(EBDigiCollection& other) {this->EcalDigiCollection::swap(other);}
-  void push_back(const Digi& digi){ DataFrameContainer::push_back(digi.id(), digi.frame().begin()); }
```

Comment:
why are these functions removed? seems unrelated to the scope of this PR, and causes build errors